ui/app/components/app-item/deployment.hbs

```diff
@@ -3,6 +3,11 @@
     <p>
       <b class="badge badge--version">v{{@deployment.sequence}}</b>
       <code>{{@deployment.id}}</code>
+      {{#if (eq @deployment.state 4)}}
+        <b class="badge badge--destroyed">
+          {{inline-svg "trash" class="icon"}}&nbsp;destroyed
```

Comment:
Just a question here but something about the lowercase surprised, did you look at it with `Destroyed`?